src/gmail.js

```diff
@@ -213,22 +213,25 @@ var Gmail_ = function(localJQuery) {
       bx_scfs = ['bx_scfs','false'];
     }
      return (bx_scfs[1] == 'true' ) ? true : false;
-  }
+  };
 
 
   api.check.is_google_apps_user =function() {
     var email = api.get.user_email();
     return email.indexOf('gmail.com', email.length - 'gmail.com'.length) == -1;
   };
 
+  api.check.is_in_popout = function() {
+    return api.tools.parse_url(window.location.href).view === 'btop';
+  };
 
```

Comment:
This method should also be added to the documentation (and if possible also to the TypeScript type declarations).